cfme/tests/intelligence/reports/test_validate_chargeback_report.py

```diff
@@ -213,21 +213,54 @@ def resource_usage(vm_ownership, appliance, provider):
     # are capturing C&U data and forcing hourly rollups by running these commands through
     # the Rails console.
 
-    command = ('Metric::Targets.perf_capture_always = {:storage=>true, :host_and_cluster=>true};')
-    appliance.ssh_client.run_rails_command(command, timeout=None)
-
-    logger.info('capturing PERF data for VM {} running on {}'.format(vm_name, provider.name))
-    appliance.ssh_client.run_rails_command(
-        "\"vm = Vm.where(:ems_id => {}).where(:name => {})[0];\
-        vm.perf_capture('realtime',1.hour.ago.utc, Time.now.utc);\
-        vm.perf_rollup_range(1.hour.ago.utc, Time.now.utc,'realtime'))\"".
-        format(provider_id, repr(vm_name)))
+    def verify_records_metrics_table(appliance, provider):
+        # Verify that rollups are present in the metric_rollups table.
+        vm_name = provider.data['cap_and_util']['chargeback_vm']
+
+        ems = appliance.db.client['ext_management_systems']
+        metrics = appliance.db.client['metrics']
+
+        rc, out = appliance.ssh_client.run_rails_command(
+            "\"vm = Vm.where(:ems_id => {}).where(:name => {})[0];\
+            vm.perf_capture('realtime', 1.hour.ago.utc, Time.now.utc)\""
+            .format(provider_id, repr(vm_name)))
```

Comment:
you use provider_id variable from outside. Please add "id" property to BaseProvider and use provider instead.